routes/tasks.js

```diff
@@ -35,6 +35,8 @@ const enableDevModeMiddleware = (req, res, next) => {
 
 router.get("/", getTasksValidator, cacheResponse({ invalidationKey: ALL_TASKS, expiry: 10 }), tasks.fetchTasks);
 router.get("/self", authenticate, tasks.getSelfTasks);
+router.get("/v1/:userId", authenticate, tasks.getTasksByUser);
```

Comment:
What is the route name?

Reply:
sorry sir didnt get you 
if you are talking about the v1 that was advised by Ankush to add v1

Reply:
how will the route name look then?

Reply:
tasks/v1/:userid

Reply:
This is not the right way/place to add versioning, please refer to some docs or google and update it accordingly

Reply:
Sir i looked into it 
i saw some examples where this is the same pattern being used